### PR TITLE
Update cert table to use v4 of scs-compatible iaas, for all the older…

### PR DESCRIPTION
**Pull request by mbuechse**
… versions have expired

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ This is a list of clouds that we test on a nightly basis against our `scs-compat
...
...GmbH               | &nbsp;<br />![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-pco-prod1-v4.yml?label=compliant)<br />![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-pco-prod2-v4.yml?label=compliant)<br />![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-pco-prod3-v4.yml?label=compliant)<br />![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-pco-prod4-v4.yml?label=compliant) | &nbsp;<br />[HM1](https://health.prod1.plusserver.sovereignit.cloud:3000/d/9ltTEmlnk/openstack-health-monitor2?orgId=1&var-mycloud=plus-pco)<br />[HM2](https://health.prod1.plusserver.sovereignit.cloud:3000/d/9ltTEmlnk/openstack-health-monitor2?orgId=1&var-mycloud=plus-prod2)<br />[HM3](https://health.prod1.plusserver.sovereignit.cloud:3000/d/9ltTEmlnk/openstack-health-monitor2?orgId=1&var-mycloud=plus-prod3)<br />[HM4](https://health.prod1.plusserver.sovereignit.cloud:3000/d/9ltTEmlnk/openstack-health-monitor2?orgId=1&var-mycloud=plus-prod4) |
```
Comment (berendt) [inline README.md:13]: https://github.com/SovereignCloudStack/standards/pull/603 is required to fix the label for check-pco-prod1-v4.

Comment (mbuechse) [reply README.md:13]: @berendt Thanks for getting going! I merged it and rebased this one. Thanks again!

**Comment by mbuechse** [comment]
Just FYI: I'm in the process of connecting to people from multiple clouds and asking whether they want to achieve compliance with v4.

**Comment by berendt** [comment]
REGIO.cloud is already v4 compatible. Nothing to do there.

**Comment by mbuechse** [comment]
> REGIO.cloud is already v4 compatible. Nothing to do there.

Yes, I did see that ;)

**Review by gtema — APPROVED** [review]

**Comment by berendt** [comment]
Looks like https://cnds.io is offline?

**Comment by mbuechse** [comment]
@berendt I think we can fitx that with #653 -- once I have rebase that one

**Pull request merged**